inputFiles/singlePhaseFlow/vtk/3D_10x10x10_compressible_vtk_hex_gravity_smoke.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" ?>
```

Comment:
Do we really want the word `vtk` in the file name?  The pamela versions of these files have already been removed or are you adding some integratedTests ?

Reply:
I've removed `vtk` from the file name.
Currently I do not add or remove integrated test; it will be in a follow-up PR.